packages/core/package.json

```diff
@@ -29,7 +29,7 @@
     "@envelop/graphql-jit": "^1.1.1",
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
-    "@faststore/api": "^2.0.97-alpha.0",
+    "@faststore/api": "*",
```

Comment:
Hmm, looks like it's getting the 1.12.38 version in `yarn.lock`. Shouldn't we use the evergreen version?

Reply:
Yeah, it's wrong, I fixed